packages/tag/src/Tag/Tag.tsx

```diff
@@ -50,7 +50,7 @@ const Root = <T extends TgphElement>({
         display="inline-flex"
         align="center"
         rounded="3"
-        pl="2"
+        pl={size === "0" ? "1" : "2"}
```

Comment:
Can we add this to a map in Tags.constants instead? Ideally all of these are configuration based and not inline prop logic. 

```
const rootSpacing = {
"0": "1",
"1": "2",
....
"
```